Makefile

```diff
@@ -77,9 +77,11 @@ $(GOBIN)/goveralls: | $(BASE) ; $(info  building goveralls...)
 
 .PHONY: lint
 lint: | $(BASE) $(GOLINT) ; $(info  running golint...) @ ## Run golint
+	$Q mkdir -p $(BASE)/test
```

Comment:
we should move the go golangci-lint so what is to point on fixing this. (golangci-lint can be in a different PR)

Reply:
teeing to /dev/stderr fails for non-root privileges,
one of the goals is to make this work as non-root privileges so i just tee to some temp file cat it and delete it.

this is not related to the lint.

Reply:
btw GOLINT is already golangci-lint. (just with default config)
`#37: GOLINT = $(GOBIN)/golangci-lint`